packages/ag-charts-community/src/chart/interaction/animationManager.ts

```diff
@@ -53,6 +55,18 @@ export class AnimationManager extends BaseManager<AnimationEventType, AnimationE
 
         const id = opts.id ?? Math.random().toString();
 
+        if (opts.throttleId) {
+            const throttleTime = this.throttles.get(opts.throttleId);
+            const isThrottled = throttleTime && opts.duration && Date.now() - throttleTime < opts.duration;
+
+            if (isThrottled) {
+                opts.delay = 0;
+                opts.duration = 1;
+            }
+
+            this.throttles.set(opts.throttleId, Date.now());
+        }
+
```

Comment:
This is the approach from before the refactor, but if you think there is a better way to match the changes let me know @iMoses